components/Api/Stability/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { FormattedMessage } from 'react-intl';
+import styles from './index.module.scss';
+import type { PropsWithChildren, FC } from 'react';
+
+type StabilityProps = PropsWithChildren<{ stability: number }>;
+
+const getStabilityClass = (stability: number) => {
+  const style = styles[`stabilityLevel${stability}`];
+  if (!style) throw new Error(`Unknown stability level: ${stability}`);
+};
```

Comment:
Don't we need a `return style` at the end of the function?
```suggestion
const getStabilityClass = (stability: number) => {
  const style = styles[`stabilityLevel${stability}`];
  if (!style) throw new Error(`Unknown stability level: ${stability}`);
  return style;
};
```